lms/djangoapps/certificates/views/webview.py

```diff
@@ -34,7 +34,9 @@
     get_certificate_url,
     emit_certificate_event,
     has_html_certificates_enabled,
-    get_certificate_template
+    get_certificate_template,
+    get_certificate_header,
+    get_certificate_footer
```

Comment:
nit: comma at the end of this line would make the diff clean when some one adds a new item in future.